gdnative-core/src/dictionary.rs

```diff
@@ -268,30 +524,30 @@ godot_test!(test_dictionary {
     let x = Variant::from_i64(42);
     let y = Variant::from_i64(1337);
 
-    let mut dict = Dictionary::new();
+    let dict = Dictionary::<Unique>::new();
```

Comment:
Interesting. `new` is only defined for `Dictionary<Unique>`, so shouldn't inference work here?

Reply:
It probably should? I can try that again without the explicit parameter.